easybuild/easyblocks/c/cp2k.py

```diff
@@ -207,6 +209,14 @@ def configure_step(self):
             options['LIBS'] += ' -lplumed'
             options['DFLAGS'] += ' -D__PLUMED2'
 
+        # ELPA
+        if self.cfg["elpa"]:
```

Comment:
@damianam I know you modelled this after the `PLUMED` support, but I'm now unsure why we actually introduced the `plumed` configure parameter at all (@wpoely86, any ideas?)...

Instead, why not go with this approach:

```python
elpa = get_software_root('ELPA')
if elpa:
    options['LIBS'] += ' -lelpa'
    options['DFLAGS'] += ' -D__ELPA3'
    options['FCFLAGSOPT'] += ' -I%s ' % os.path.join(elpa,'include/elpa-%s/modules' % get_software_version('ELPA'))